diff --git a/meson.build b/meson.build
--- a/meson.build
+++ b/meson.build
@@ -90,6 +90,9 @@ conf = configuration_data()

 conf.set_quoted('GETTEXT_PACKAGE', meson.project_name())
 conf.set_quoted('LOCALE_DIR', gkr_prefix / get_option('localedir'))
+conf.set('HAVE_GETTEXT', true)
+conf.set('ENABLE_NLS', true)
+conf.set_quoted('GNOMELOCALEDIR', gkr_prefix / get_option('localedir'))
 conf.set_quoted('VERSION', meson.project_version())
 conf.set_quoted('LIBGCRYPT_VERSION', libgcrypt_dep.version())
 conf.set('HAVE_LIBCAPNG', libcap_ng_dep.found())
